src/context/transaction/inputValidations.js

```diff
@@ -13,9 +13,18 @@ const isPositiveNumber = (input) => {
   }
   return input > 0;
 };
-// Validate Transaction amount is two decimals max
 
-// Validate Transaction fee is two decimals max
+// Validate number is two decimals max
+const isNumberWithMaxTwoDecimals = (input) => {
+  if (typeof input !== "number" || isNaN(input)) {
+    return false;
+  }
+  const inputString = input.toString();
+  // const [whole, decimal] = inputString.split(".");
+  const decimal = inputString.split(".")[1];
+  return !decimal || (!!decimal && decimal.length <= 2);
```

Comment:
it's hard to follow what this means. so, in this context, i believe using variables might help a lot with readability.

Reply:
Agree that would take some noodling in the future. New version is much clearer.